modules/social_features/social_event/config/install/field.storage.node.field_event_image.yml

```diff
@@ -23,7 +23,7 @@ settings:
 module: image
 locked: false
 cardinality: 1
-translatable: true
+translatable: false
```

Comment:
Images were made translatable on purpose. An Image could contain text so it may be desirable to upload a different image in one language than in another.